### ISSUE TITLE
open english model

### ISSUE THREAD

**Issue by gedonglai** [tdozat/Parser#10]
Hello，
         Do I need to train a english model for myself?  there is no ready-made english model for parsing?

**Comment by allanj** [comment]
I totally agree with you. Though many papers produce dependency parsing results. I think it is better to open source the trained model. I think many people now are still using Stanford parser or spacy parser
